syntax = "proto3";

package EsPb;

message HeroEquipRecommendation {
    repeated sHeroSetEquip heroSetEquipList = 1;
}

message sHeroSetEquip {
    repeated int32 setEffectNo = 1;
    int32 count = 2;
}